tensorflow/core/kernels/gpu_utils.cc

```diff
@@ -19,14 +19,14 @@ limitations under the License.
 
 #include <iterator>
 
-#include "google/protobuf/any.pb.h"
 #include "absl/algorithm/container.h"
+#include "google/protobuf/any.pb.h"
```

Comment:
Blank line should separate tensorflow/ imports

Reply:
Sounds good.